src/styles/sidebar/components/annotation-document-info.scss

```diff
@@ -0,0 +1,14 @@
+.annotation-document-info {
+  font-size: 13px;
```

Comment:
I know that our typography stack is not well defined at the moment, but I think it would still be preferable to use the font-size variables that do exist from the "Typography" section in `variables.scss` rather than hard-coding them here. That will help us establish at least which things are supposed to be the same size and adjust them uniformly.